app/views/users/show.html.erb

```diff
@@ -10,5 +10,6 @@
         </li>
     <% end %>
     <br/>
+    <br/>
     <%= link_to 'show more', user_posts_path(@user.id) %>
 </ul>
```

Comment:
- [ ] You're required to display user's 3 most recent posts on the user show  page. Kindly attend to that.

### Area of attention :point_down: 

![recent-post](https://github.com/BenM4k/BlogApp/assets/26597432/d5fe16f4-f7fa-4dce-81c7-7c367907da99)

*Hint: * It should look something similar to this :point_down: 
```
  <% @user.recent_post.each do |post| %>

<h4><%= link_to post.title, user_post_path(post) %></h4>
<p><%= post.text %></p>
<p>
<small>Comments: <%= post.comments_counter || 0 %>,</small> 
<small>Likes: <%= post.likes_counter || 0 %></small>
</p>

<% end %>

```